README.md

```diff
@@ -33,7 +33,7 @@ Run the following commands from within `wordpress` folder.
 Do a one time setup for application CRD:
 
 ```shell
-make crd/install
+kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
```

Comment:
we you replace this `make app/doctor`? It also makes sure that the environments are properly setup.

Reply:
Done.